py/server/deephaven/table.py

```diff
@@ -816,6 +816,16 @@ def flatten(self) -> Table:
         """Returns a new version of this table with a flat row set, i.e. from 0 to number of rows - 1."""
         return Table(j_table=self.j_table.flatten())
 
+    def remove_blink(self) -> Table:
+        """Returns a new version of this table without specialized blink table aggregation semantics."""
+        if self.is_blink:
+            try:
+                return Table(j_table=self.j_table.removeBlink())
+            except Exception as e:
+                raise DHError(e, "failed to remove blink table semantics.") from e
+        else:
+            raise RuntimeError("Table is not a blink table, so blink table semantics cannot be removed.")
```

Comment:
Should this:
1. Error (like here)?
2. return self?

I'm not sure which provides the most natural behavior in the wild or what is most consistent with other API operations we have.  I might guess 2 is more consistent, but I'm not sure.
@rcaudy @jmao-denver @rbasralian

Reply:
I think it should just return `self.j_table.removeBlink()` without 'thinking about it' (i.e. remove the `if` and lean on the behavior of the Java implementation, which the javadocs say happens to be option 2: `@return A non-blink child table, or this table if it is not a blink table`)